pengwin-setup.d/fcitx.sh

```diff
@@ -79,7 +79,7 @@ if (whiptail --title "fcitx" --yesno "Would you like to install fcitx for improv
 
 	if (whiptail --title "fcitx-autostart" --yesno "Would you like fcitx-autostart to run each time you open Pengwin? WARNING: Requires an X server to be running or it will generate errors." 9 70) then
 		echo "Placing fcitx-autostart in /etc/profile.d/fcitx"
-		sudo sh -c 'echo "fcitx-autostart &>/dev/null" >> /etc/profile.d/fcitx'
+		sudo sh -c 'echo "dbus-launch /usr/bin/fcitx-autostart > /dev/null 2>&1" >> /etc/profile.d/fcitx'
```

Comment:
After reading the last comment about fixing dbus. We can do not apply this change but adding .sh to the file so the line is:

`sudo sh -c 'echo "/usr/bin/fcitx-autostart > /dev/null 2>&1" >> /etc/profile.d/fcitx.sh'`